spec/default_module_facts.yml

```diff
@@ -1,2 +1,4 @@
 ---
-ipaddress6: '::'
+ipaddress6: '2001:db8::c0:ffee'
+networking:
+  ip6: '2001:db8::c0:ffee'
```

Comment:
Nested don't facts work and you'll break things because it doesn't do a deep merge. We have a helper `override_facts`: https://github.com/voxpupuli/voxpupuli-test#fact-handling

I would really like to get rid of this file. If we need custom facts, just call it as native Ruby.

Something like this may work:
```ruby
add_custom_fact :ipaddress6, '2001:db8::c0:ffee'
add_custom_fact: :networking, lambda { |_os, facts| facts[:networking]['ip6'] = facts[:ipaddress6] }
```

But really, I think that if a test cares about IPv6 it should use:
```ruby
context 'with IPv6' do
  let:(facts) { override_facts(super(), ipaddress6: '2001:db8::c0:ffee', networking: { ip6: '2001:db8::c0:ffee' }) }
end
```

Reply:
hmm.. I just realized it works fine without `networking.ip6` override.. I guess facterdb does the magic. So we can just drop it (and then drop the legacy `ipaddress6` in next batch)

Reply:
IMHO a good test doesn't rely on FacterDB to ensure there's an IPv6 address (or is absent) when asserting the IPv6 code works well.

Reply:
IPv6 address is added to the facts explicitly now. There is unit test to ensure behavior is good w/o any IPv6 defined.